### ISSUE TITLE
Use ghost batch normalization with slim

### ISSUE THREAD

**Issue by ohadlights** [tensorflow/tensorflow#18893]
Posted in Stack Overflow:
https://stackoverflow.com/questions/49967489/use-ghost-batch-normalization-with-slim

### Details

OS Platform and Distribution: N/A
TensorFlow installed from: N/A
TensorFlow version: 1.8
Bazel version: N/A
CUDA/cuDNN version: N/A
GPU model and memory: N/A
Exact command to reproduce: use slim.batch_norm

### Describe the problem

normalization_layers.BatchNormalization takes an argument virtual_batch_size that tells the layer to use "Ghost Batch Normalization", which creates virtual sub-batches which are each normalized separately.

slim.batch_norm doesn't expose this argument. is there a way to use ghost BN with slim?

Thanks.

**Comment by tensorflowbutler** [comment]
Thank you for your post. We noticed you have not filled out the following field in the issue template. Could you update them if they are relevant in your case, or leave them as N/A? Thanks.
Have I written custom code
OS Platform and Distribution
TensorFlow installed from
TensorFlow version
Bazel version
CUDA/cuDNN version
GPU model and memory
Exact command to reproduce

**Comment by ohadlights** [comment]
Added 'Details' section. thanks.

**Comment by tensorflowbutler** [comment]
Nagging Assignee @poxvoculi: It has been 14 days with no activity and this issue has an assignee. Please update the label and/or status accordingly.

**Comment by poxvoculi** [comment]
This question is better asked on [StackOverflow](http://stackoverflow.com/questions/tagged/tensorflow) since it is not a bug or feature request. There is also a larger community that reads questions there.

If you think we've misinterpreted a bug, please comment again with a clear explanation, as well as all of the information requested in the [issue template](https://github.com/tensorflow/tensorflow/issues/new). Thanks!

**Comment by ohadlights** [comment]
I think it's actually a feature request :)

The request is to add a virtual_batch_size parameter to slim.batch_norm.

**Comment by tensorflowbutler** [comment]
Please remove the assignee, as this issue is inviting external contributions. Otherwise, remove the `contributions welcome` label. Thank you.

**Comment by faintedremix** [comment]
Should fused_batch_norm() also contain this feature?

**Comment by FollowKenny** [comment]
Hey, is this feature request still pending ? I would like to tackle it if you guys think it is suitable as a first contribution to tensorflow.

**Comment by mohantym** [comment]
Hi @ohadlights ! 1.x version are not supported any more. You can use this feature from [tf.compat.v1.keras.layers.BatchNormalization ](https://www.tensorflow.org/api_docs/python/tf/compat/v1/keras/layers/BatchNormalization) in 2.x versions. Thank you!
